src/modules/home/ui/components/navbar.tsx

```
@@ -0,0 +1,120 @@
+        {navbarItems.map((item) => (
+          <NavbarItem
+            key={item.href}
+            href={item.href}
+            isActive={pathname === item.href}
+          >
+            {item.children}
+          </NavbarItem>
+        ))}
+      </div>
+
+      {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)